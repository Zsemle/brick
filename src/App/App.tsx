import React from 'react'
import './App.css'
import Footer from '../Footer/Footer'
import Header from '../Header/Header'
import Content from '../Content/Content'
import FilterPage from '../FilterPage/FilterPage'

const App:React.FunctionComponent = function ():JSX.Element {
  return (
    <div>
      <Header/>
      <Content>
        <FilterPage/>
      </Content>
      <Footer/>
    </div>
  )
}

export default App
