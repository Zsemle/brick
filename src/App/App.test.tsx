import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import App from './App';

describe('<App />', () => {
  it('renders the app', () => {
    const wrapper:ShallowWrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
