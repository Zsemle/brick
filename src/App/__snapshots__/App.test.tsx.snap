// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders the app 1`] = `
<div>
  <Header />
  <Content>
    <FilterPage />
  </Content>
  <Footer />
</div>
`;
